cluster/aws/config-default.sh

```diff
@@ -81,8 +81,13 @@ MASTER_IP_RANGE="${MASTER_IP_RANGE:-10.246.0.0/24}"
 # If set to auto, a new Elastic IP will be acquired
 # Otherwise amazon-given public ip will be used (it'll change with reboot).
 MASTER_RESERVED_IP="${MASTER_RESERVED_IP:-}"
+
+# Runtime config
 RUNTIME_CONFIG="${KUBE_RUNTIME_CONFIG:-}"
-ENABLE_EXPERIMENTAL_API="${KUBE_ENABLE_EXPERIMENTAL_API:-false}"
+
+# Enable various v1beta1 features
+ENABLE_DEPLOYMENTS="${KUBE_ENABLE_DEPLOYMENTS:-}"
+ENABLE_DAEMONSETS="${KUBE_ENABLE_DAEMONSETS:-}"
```

Comment:
@nikhiljindal See these lines. They need to be replicated in config-test I think. I'll move them.